e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -76,6 +76,22 @@ Scenario('HMCTS admin uploads standard directions with other documents', async (
   I.seeAnswerInTab('2', 'Other documents 2', 'Upload a file', 'mockFile.txt');
 });
 
+Scenario('HMCTS admin uploads C2 documents to the case', async (I, caseViewPage, uploadC2DocumentsEventPage) => {
+  await caseViewPage.goToNewActions(config.administrationActions.uploadC2Documents);
+  uploadC2DocumentsEventPage.uploadC2Document(config.testFile, 'Rachel Zane C2');
+  await I.completeEvent('Save and continue');
+  I.seeEventSubmissionConfirmation(config.administrationActions.uploadC2Documents);
+  await caseViewPage.goToNewActions(config.administrationActions.uploadC2Documents);
+  uploadC2DocumentsEventPage.uploadC2Document(config.testFile, 'Jessica Pearson C2');
+  await I.completeEvent('Save and continue');
+  I.seeEventSubmissionConfirmation(config.administrationActions.uploadC2Documents);
+  caseViewPage.selectTab(caseViewPage.tabs.documents);
+  I.seeAnswerInTab('1', 'C2 1', 'Upload a file', 'mockFile.txt');
```

Comment:
Shouldn't we assert the additional fields? Just to ensure they are displayed, date might be trickier but author should be easy here. What do you think?

Reply:
We are testing it in the controller, I think I am fine with not checking it here, but not sure.. what do you think?

Reply:
Hmmm I can see you're point but i'm not overly keen on testing these in e2e! I think at this point we should be asserting that documents where generated and leave meta data for int tests. In saying that should we still assert for description?